### PR TITLE
Remove skips from doctests (4 of 6)

### PR DESCRIPTION
**Pull request by zzhengnan**
This is one of 6 PRs that collectively addresses #7823. Please see https://github.com/dask/dask/issues/7823#issuecomment-873414398 for more context.

Links to all 6 PRs
- #7874
- #7873
- #7872
- #7865
- #7864
- #7863

### CODE REVIEW

File: dask/bag/core.py [hunk 1]
```diff
@@ -1928,18 +1928,19 @@ def bag_zip(*bags):
...
 
     When what you really wanted was more along the lines of the following:
 
-    >>> list(fizzbuzzz) # doctest: +SKIP
-    [(0, 0), (3, None), (None, 5), (6, None), (None 10), (9, None),
-    (12, None), (15, 15), (18, None), (None, 20), (None, 25), (None, 30)]
+    >>> list(fizzbuzz) # doctest: +SKIP
+    (0, 0), (3, None), (None, 5), (6, None), (9, None), (None, 10),
+    (12, None), (15, 15), (18, None), (None, 20),
+    (21, None), (24, None), (None, 25), (27, None), (30, 30)
```
Comment (zzhengnan) [inline dask/bag/core.py:1943]: Re the following comment, it's not clear to me what the original author was going for, so I took my best guess.
> When what you really wanted was more along the lines of the following

File: dask/bag/random.py [hunk 2]
```diff
@@ -21,11 +21,10 @@ def sample(population, k):
...
     --------
-    >>> import dask.bag as db # doctest: +SKIP
-    ... from dask.bag import random
-    ...
-    ... b = db.from_sequence(range(5), npartitions=2)
-    ... list(random.sample(b, 3).compute())
+    >>> import dask.bag as db
+    >>> from dask.bag import random
+    >>> b = db.from_sequence(range(5), npartitions=2)
+    >>> list(random.sample(b, 3).compute())  # doctest: +SKIP
```
Comment (zzhengnan) [inline dask/bag/random.py:27]: Moving the SKIP to the _actual_ line that we want to skip feels more intuitive in my opinion.

**Comment by zzhengnan** [comment]
@jsignell Ready for review.

File: dask/bag/core.py (hunk 1 above)
Comment (jsignell) [reply dask/bag/core.py:1945]: huh yeah. I am not sure either. I guess what you have is fine but just wrap it in square brackets like the original.

Comment (zzhengnan) [reply dask/bag/core.py:1945]: Will do. 

Only problem is I'd accidentally deleted my fork so can't push changes to the feature branch. As seen below, the feature branch is grayed out and can no longer be clicked on (normally clicking on it takes you to the fork). This applies to #7864 as well.
image
 
Any idea what I can do here without having to open a _new_ PR? I do still have access to the fork locally so no work is lost but I'd prefer not to open a new PR if possible.

Comment (jsignell) [reply dask/bag/core.py:1945]: Oh wierd. I am going to see if I can still push.

Comment (jsignell) [reply dask/bag/core.py:1945]: Nope.

Comment (jsignell) [reply dask/bag/core.py:1945]: I'm just going to merge this since the last comment is about a skipped doctest output.

**Pull request merged**